torch/distributed/_symmetric_memory/__init__.py

```diff
@@ -110,6 +110,8 @@ def get_symm_mem_workspace(group_name: str, min_size: int) -> _SymmetricMemory:
         _SymmetricMemory: the symmetric memory workspace associated with the
         group.
     """
+    enable_symm_mem_for_group(group_name)
```

Comment:
Ah this looks like a key change! Does it mean that now, if one calls the fused matmul+comms ops, they lazily initialize the SymmetricMemory on their own?

Reply:
Ah, right! I realized it's redundant to require users to call `enable_symm_mem_for_group(group_name)` when they are already explicitly using the group for symmetric memory. This indirectly addresses the tracing issue you encountered.